developer-workflow/stdlib.rst

```diff
@@ -137,3 +138,17 @@ once the authors of the module sign
 :ref:`contributor agreements <contributor_agreement>`.
 
 .. _issue tracker: https://github.com/python/cpython/issues
+
+Adding a new env var
+--------------------
+
+From Python 3.13 onwards, name environment variables with underscores for
+readability and accessibility.
+
+For example, use ``PYTHON_CPU_COUNT`` instead of ``PYTHONCPUCOUNT``.
+
+See also:
+
+* :ref:`python:using-on-envvars`
+* `Discourse discussion
+  <https://discuss.python.org/t/change-environment-variable-style/35180>`__
```

Comment:
```suggestion
* `"Change environment variable style" Discourse discussion
  <https://discuss.python.org/t/change-environment-variable-style/35180>`__
```

Even if in context it's understandable, it's better to avoid generic link text.